es-bs-base/scss/_variables.scss

```diff
@@ -1511,26 +1511,26 @@ $form-feedback-tooltip-border-radius: $tooltip-border-radius !default;
 // Popovers
 
 $popover-font-size:                 $font-size-sm !default;
```

Comment:
The font size within the popover body should be 16, not 14. I think the one you want is `$font-size-100`.